lib/internal/url.js

```diff
@@ -649,10 +649,49 @@ function getObjectFromParams(array) {
 
 class URLSearchParams {
   constructor(init = '') {
-    if (init instanceof URLSearchParams) {
-      const childParams = init[searchParams];
-      this[searchParams] = childParams.slice();
+    if (init === null || init === undefined) {
```

Comment:
The user can do `new URLSearchParams(undefined)`. The default is there to echo the [`= ""`](https://url.spec.whatwg.org/#dom-urlsearchparams-urlsearchparams-init-init) in the spec, though now I agree it's a bit pointless.